hack/local-up-karmada.sh

```diff
@@ -48,6 +49,16 @@ else
   echo "not pass"
   util::install_tools "sigs.k8s.io/kind" $kind_version
 fi
+
+echo -n "Preparing: 'kwokctl' existence check - "
+if util::cmd_exist kwokctl; then
+  echo "passed"
+else
+  echo "not pass"
+  wget -O /usr/local/bin/kwokctl "https://github.com/kubernetes-sigs/kwok/releases/download/v0.0.1/kwokctl-$(go env GOOS)-$(go env GOARCH)" && \
```

Comment:
`/usr/local/bin` directory is not always permitted. Some guys run without `root`